### PR TITLE
[DOCS] Updates Watcher examples for code testing

### PR DESCRIPTION
**Pull request by lcawl**
Related to https://github.com/elastic/elasticsearch/issues/30665

This PR adds the appropriate testing keywords for all of the x-pack/docs/en/watcher pages, so that they can be removed from the build.gradle file.

Note that most of these were examples that contained just a subsection of a watch definition, so I marked them as //NOTCONSOLE.  If you would prefer to develop these snippets into full examples so that they can be automatically tested, let me know and I'll help.

### CODE REVIEW

File: x-pack/docs/en/watcher/example-watches/example-watch-meetupdata.asciidoc [hunk 1]
```diff
@@ -65,8 +71,11 @@ To set up the watch:
     }
   },
 --------------------------------------------------
+// NOTCONSOLE
+--
 . Load data into the watch payload by creating an input that searches the meetup data for events that have _Open Source_ as a topic. You can use aggregations to group the data by city, consolidate references to the same events, and sort the events by date.
 +
```
Comment (spinscale) [inline x-pack/docs/en/watcher/example-watches/example-watch-meetupdata.asciidoc]: leftovers or asciidoc features I am not aware off?

File: x-pack/docs/en/watcher/example-watches/example-watch-meetupdata.asciidoc [hunk 2]
```diff
@@ -135,19 +144,28 @@ To set up the watch:
...
   },
 --------------------------------------------------
+// NOTCONSOLE
 <1> Elasticsearch Date math is used to select the Logstash indices that contain the meetup data. The second pattern is needed in case the previous hour crosses days.
 <2> Find all of the RSVPs with `Open Source` as a topic.
 <3> Group the RSVPs by city.
 <4> Consolidate multiple RSVPs for the same event.
 <5> Sort the events so the latest events are listed first.
 <6> Group the events by name.
+--
```
Comment (spinscale) [inline x-pack/docs/en/watcher/example-watches/example-watch-meetupdata.asciidoc]: I assume that belongs to the two lines below?

File: x-pack/docs/en/watcher/example-watches/example-watch-meetupdata.asciidoc [hunk 3]
```diff
@@ -135,19 +144,28 @@ To set up the watch:
...
 
 . To determine whether or not there are any Open Source events, add a compare condition that checks the watch payload to see if there were any search hits.
++
+--
 [source,js]
 --------------------------------------------------
 "compare" : { "ctx.payload.hits.total" : { "gt" : 0 }}
 --------------------------------------------------
+// NOTCONSOLE
+--
```
Comment (spinscale) [inline x-pack/docs/en/watcher/example-watches/example-watch-meetupdata.asciidoc]: If this does not belong to the `//NOTCONSOLE` part, would it make sense to add a newline here?

**Review by spinscale — APPROVED** [review]
I left three comments regarding asciidoc format, that are different than other snippets, but look like many others, so maybe I am just missing out asciidoc features here. 

Otherwise LGTM

File: x-pack/docs/en/watcher/example-watches/example-watch-meetupdata.asciidoc (hunk 1 above)
Comment (lcawl) [reply x-pack/docs/en/watcher/example-watches/example-watch-meetupdata.asciidoc]: If you mean the + and -- around the code example, it's just a way to ensure it stays nested in the list. Not really necessary in this case.

File: x-pack/docs/en/watcher/example-watches/example-watch-meetupdata.asciidoc (hunk 2 above)
Comment (lcawl) [reply x-pack/docs/en/watcher/example-watches/example-watch-meetupdata.asciidoc]: Ditto

File: x-pack/docs/en/watcher/example-watches/example-watch-meetupdata.asciidoc (hunk 3 above)
Comment (lcawl) [reply x-pack/docs/en/watcher/example-watches/example-watch-meetupdata.asciidoc]: Ditto

**Pull request merged**